### DISCUSSION TITLE
Toggle Sticky not toggling Float

### DISCUSSION THREAD

**Discussion by tbsmn** [asmvik/yabai#745] [Q&A] [answered]
Hi everybody I have this Problem: When I toggle a window to be sticky `yabai -m window --toggle sticky` and the window was tiled ist is changed to float. But when I toggle it back it stays in float-mode and is not switching to tiled.

Is there a workaround for this?
Version is 3.3.4

**Comment by dominiklohmann** [comment, marked as answer]
This would need to be implemented within yabai. Setting `sticky` should not set `floating`, but `floating` should be implied if `sticky` is set. That way you do not lose information on the way bug.

I would actually consider this a bug in the current implementation.

**Reply by dominiklohmann** [reply]
I have created an issue accordingly, see https://github.com/koekeishiya/yabai/issues/760

**Reply by tbsmn** [reply]
Thank You!
